### PR TITLE
Performance: Adds includes compiler optimize option

### PR DESCRIPTION
**Pull request by kirankumarkolli**
Performance: Adds includes compiler optimize option

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Microsoft.Azure.Cosmos.csproj [hunk 1]
```diff
@@ -39,6 +39,7 @@
     <IncludeSource>false</IncludeSource>
     <RootNamespace>Microsoft.Azure.Cosmos</RootNamespace>
     <NoWarn>NU5125</NoWarn>
+    <Optimize Condition="'$(Platform)'=='Release'">true</Optimize>
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/src/Microsoft.Azure.Cosmos.csproj:42]: This should be using `Configuration` not `Platform` right?

`<Optimize Condition="'$(Configuration)'=='Release'">true</Optimize>`

File: Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/CosmosBenchmark.csproj [hunk 2]
```diff
@@ -4,7 +4,8 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <RootNamespace>CosmosBenchmark</RootNamespace>
     <AssemblyName>CosmosBenchmark</AssemblyName>
-    <ServerGarbageCollection>true</ServerGarbageCollection>    
+    <ServerGarbageCollection>true</ServerGarbageCollection>
+    <Optimize Condition="'$(Platform)'=='Release'">true</Optimize>
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos.Samples/Tools/Benchmark/CosmosBenchmark.csproj:8]: `<Optimize Condition="'$(Configuration)'=='Release'">true</Optimize>` ?

File: Microsoft.Azure.Cosmos/src/Microsoft.Azure.Cosmos.csproj (hunk 1 above)
Comment (ealsur) [reply Microsoft.Azure.Cosmos/src/Microsoft.Azure.Cosmos.csproj:42]:
```suggestion
    <Optimize Condition="'$(Configuration)'=='Release'">true</Optimize>
```

**Comment by kirankumarkolli** [comment]
@ealsur haha such a basic mistake. Thank for catching it.

**Pull request merged**